war/src/main/webapp/scripts/hudson-behavior.js

```diff
@@ -1901,8 +1901,11 @@ function xor(a, b) {
 // eslint-disable-next-line no-unused-vars
 function replaceDescription(initialDescription, submissionUrl) {
   var d = document.getElementById("description");
-  d.firstElementChild.nextElementSibling.innerHTML =
-    "<div class='jenkins-spinner'></div>";
+  let button = d.firstElementChild.nextElementSibling;
```

Comment:
```
hudson-behavior.js:1904 Uncaught TypeError: Cannot read properties of null (reading 'firstElementChild')
    at replaceDescription (hudson-behavior.js:1904:18)
    at HTMLAnchorElement.<anonymous> (editable-description.js:10:14)
```
when I hit edit description at `/manage/computer/(built-in)/`

Reply:
ahh the built-in has no description. Worked around by only showing the button when there is an actual div with id `description`